app/controllers/katello/api/v2/hosts_bulk_actions_controller.rb

```diff
@@ -194,12 +194,16 @@ def content_overrides
       respond_for_async :resource => task
     end
 
-    api :PUT, "/hosts/bulk/environment_content_view", N_("Assign the environment and content view to one or more hosts")
+    api :PUT, "/hosts/bulk/environment_content_view", N_("Assign the environment, content view and release versions to one or more hosts")
     param_group :bulk_params
-    param :environment_id, Integer
-    param :content_view_id, Integer
+    param :environment_id, Integer, :required => true
+    param :content_view_id, Integer, :required => true
+    param :release_version, String, :desc => N_("Release version")
     def environment_content_view
-      task = async_task(::Actions::BulkAction, ::Actions::Katello::Host::UpdateContentView, @hosts, @view.id, @environment.id)
+      ignore_release_version = !params.key?(:release_version)
+      task = async_task(::Actions::BulkAction,
+                        ::Actions::Katello::Host::UpdateContentView,
+                        @hosts, @view.id, @environment.id, params[:release_version], ignore_release_version)
```

Comment:
Why can't you just pass params[:release_version] to the UpdateContentView action and check for truthy/falseness instead of using `ignore_release_version`?